presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -1754,6 +1754,13 @@ public void testOrderByMultipleFields()
     }
 
     @Test
+    public void testOrderByDuplicateFields()
+            throws Exception
+    {
+        assertQueryOrdered("SELECT custkey, custkey FROM orders ORDER BY custkey, custkey");
```

Comment:
Add the test I mentioned in my previous comments to ensure the sort order for fields that appear multiple times is handled correctly. E.g.,

```
SELECT custkey, custkey FROM orders ORDER BY custkey ASC, custkey DESC
```